--- a/include/strong_type/arithmetic.hpp
+++ b/include/strong_type/arithmetic.hpp
@@ -163,12 +163,16 @@ namespace strong
 }
 
-template <typename T, typename Tag, typename ... Ms>
-class std::numeric_limits<strong::type<T, Tag, Ms...>>
-#if __cplusplus >= 202003L
-    requires strong::type_is<strong::type<T, Tag, Ms...>, strong::arithmetic>
-        : public std::numeric_limits<T>
+template <typename T, typename Tag, typename ... Ms>
+#if __cplusplus >= 202003L
+requires strong::type_is<strong::type<T, Tag, Ms...>, strong::arithmetic>
+#endif
+class std::numeric_limits<strong::type<T, Tag, Ms...>>
+#if __cplusplus >= 202003L
+    : public std::numeric_limits<T>
 #else
     : public std::conditional<
         strong::type_is_v<strong::type<T, Tag, Ms...>, strong::arithmetic>,
         std::numeric_limits<T>,
         std::numeric_limits<void>
     >::type
 #endif
 {
